extension/src/experiments/model/index.ts

```diff
@@ -478,6 +478,10 @@ export class ExperimentsModel extends ModelWithPersistence {
     this.persistNbOfCommitsToShow()
   }
 
+  public resetNbfCommitsToShow(branch: string) {
+    delete this.numberOfCommitsToShow[branch]
```

Comment:
I've looked over the code multiple times but I can't figure out how this change is saved in the state across sessions.  The function only touches `this.numberOfCommitsToShow` yet if I reload the window, the change is saved so the `numberOfCommitsToShow` is being persisted somewhere, right? How is it working though 😅 What am I missing?

Reply:
The memento could just be holding a reference to the original object. I will make an update.